applications/conf/routes

```diff
@@ -114,3 +114,6 @@ GET        /$leftSide<[^+]+>+*rightSide
 
 # Google site verification
 GET        /google$account<[\w\d-]*>.html                                       controllers.SiteVerificationController.googleSiteVerify(account)
+
+# Formstack form submission
+POST      /formstack-campaign/submit                                            controllers.CampaignsController.formSubmit()
```

Comment:
this is in several routes files, is this necessary?

Reply:
Good point, I assumed one was dev, one preview and one production... but let me actually check that.

Reply:
Just ftr: the route should be in the project your work runs in, and then duplicated in dev-build & preview.
Also NB: this is in the wrong directory and should be in `article` not `applications` - will change

Reply:
Yeah the routing requirements here are a bit frustrating, as there is nothing to guarantee (programmatically) that all the routing files are aligned and not diverging over time. It's a bad DX that we're thinking how to address.

Reply:
can we remove the mention of `formstack`? I don't know if we'll keep on using that product in the long term

Reply:
@nicl: I used a common routes file to share a route from a library to multiple Play projects in a previous job. Can I help?